macros/src/interface/mod.rs

```diff
@@ -0,0 +1,21 @@
+mod generate;
+mod parse;
+
```

Comment:
Shouldnt there be "use proc_macro2" or "use proc_macro2::TokenStream" here?

Reply:
No, that doesn't leak into submodules, so they would be unused imports :)